patient-connect/000_searchInstallation.md

```diff
@@ -10,42 +10,48 @@ To use the Care Search module, this block of code should be placed on the page w
 
 
 ```html
-<section id=”search”></section>
-<script type=”text/javascript” src=”https://connect.loyalhealth.com/client/search.bundle.js” data-id=”search-client-id” data-value=”[Your Client ID]” async>
-</script>
+<section id="loyal-search"></section>
+<script type="text/javascript" src="https://connect.loyalhealth.com/client/search.bundle.js" data-loyal-client-id="[Your Client ID]" data-loyal-market-id="[Your Market ID]" data-loyal-profile-id="[Website Profile ID (optional)]"></script>
 ```
 
 ### Configure routes on your server
 
-These routes need to be set up to redirect to the same page where the search module is loading:
+Routes need to be set up to redirect to the same page where the search module is loading. These are the defaults that loyal provides:
 
-`[your root url]`
-
-`[your root url]/results`
+```
+[your root url]
+[your root url]/results
+[your root url]/provider/:entityId
+[your root url]/location/:entityId
+```
+We allow for customization of routing in search settings, so if you choose to customize the routes, your routes may look different than those above. For example, it may look like this:
 
-`[your root url]/provider/:entityId`
+```
+[your root url]
+[your root url]/results
+doctors/:entityId
```

Comment:
I think urls on line 32 and 33 are ones where we remove root route rather than customizing it.

Reply:
I think referring them to the search settings is more appropriate as it gives visuals of what the routes will look like. I also couldn't come up with clear verbiage here to describe it -- do you have a suggestion for what we might say?